core/templates/domain/state_card/StateCardObjectFactory.ts

```diff
@@ -239,10 +265,12 @@ export class StateCardObjectFactory {
   createNewCard(
       stateName: string, contentHtml: string, interactionHtml: string,
       interaction: Interaction, recordedVoiceovers: RecordedVoiceovers,
-      contentId: string): StateCard {
+      writtenTranslations: WrittenTranslations, contentId: string): StateCard {
```

Comment:
Apparently sometimes null is passed in for this argument (I saw that in several files). That seems wrong, right? Shouldn't it be an empty dict/list?